packages/backend-function/src/factory.ts

```diff
@@ -121,6 +123,18 @@ export type FunctionProps = {
    * schedule: "0 9 * * 2" // every Monday at 9am
    */
   schedule?: FunctionSchedule | FunctionSchedule[];
+
+  /**
+   * Attach Lambda layers to a function
+   * @see [Lambda layers](https://docs.aws.amazon.com/lambda/latest/dg/chapter-layers.html)
```

Comment:
```suggestion
   * @see https://docs.amplify.aws/react/build-a-backend/functions/lambda-layers
   * @see [AWS documentation for Lambda layers](https://docs.aws.amazon.com/lambda/latest/dg/chapter-layers.html)
```

adds a placeholder for a new docs page in the amplify docs and makes the anchor text a more descriptive in it describes where we're navigating to

Reply:
updated both with anchor text